File: SistemaVeterinario/SistemaVeterinario/Veterinario.java
public class Veterinario extends Persona
{
    String strEspecialidad;

    public Veterinario(String strNombre, String strTelefono,
                       String strEspecialidad)
    {
        super(strNombre, strTelefono);
        this.strEspecialidad = strEspecialidad;
    }

    public void mostrarVeterinario()
    {
        System.out.println("Nombre: " + strNombre);
        System.out.println("Telefono: " + strTelefono);
        System.out.println("Especialidad: " + strEspecialidad);
    }
}

File: Codigo/SistemaVeterinario/logica/Veterinario.cs
namespace SistemaVeterinario.logica
{
    public class Veterinario : Persona
    {
        public string strEspecialidad { get; set; }

        public Veterinario(string strNombre, string strTelefono, string strEspecialidad)
            : base(strNombre, strTelefono)
        {
            this.strEspecialidad = strEspecialidad;
        }

        public void mostrarVeterinario()
        {
            Console.WriteLine("Nombre: " + strNombre);
            Console.WriteLine("Telefono: " + strTelefono);
            Console.WriteLine("Especialidad: " + strEspecialidad);
        }
    }
}